agent/engine/docker_task_engine.go

```diff
@@ -1392,11 +1392,11 @@ func (engine *DockerTaskEngine) applyContainerState(task *apitask.Task, containe
 	}
 	metadata := transitionFunction(task, container)
 	if metadata.Error != nil {
-		seelog.Infof("Task engine [%s]: error transitioning container [%s] to [%s]: %v",
-			task.Arn, container.Name, nextState.String(), metadata.Error)
+		seelog.Infof("Task engine [%s]: error transitioning container [%s (Runtime ID: %s)] to [%s]: %v",
```

Comment:
if there is error in transitioning, it could also have empty container ID..but don't want to add a logic to check states and log accordingly. this is just to keep in mind ID can be empty

Reply:
There can be empty runtime ID. But did not add any logic for that so that the code does not become cumbersome.